cli/command/run/cmd.go

```diff
@@ -103,6 +104,9 @@ func Command(cli *command.SauceCtlCli) *cobra.Command {
 	cmd.Flags().StringVar(&sauceignore, "sauceignore", "", "Specifies the path to the .sauceignore file.")
 	cmd.Flags().StringToStringVar(&experiments, "experiment", map[string]string{}, "Specifies a list of experimental flags and values")
 	cmd.Flags().BoolVarP(&dryRun, "dry-run", "", false, "Simulate a test run without actually running any tests.")
+	cmd.Flags().StringVar(&mode, "mode", "", "Specifies the environment in which the tests should run. Choice: docker|sauce.")
```

Comment:
Based the latest discussion, it sounded like we are getting rid of the `mode` override from the CLI arg perspective completely. Hence I'd suggest to remove this line altogether. Also remove `"use --mode instead"` from line 109.